packages/common/src/utils/formatting.ts

```diff
@@ -39,6 +39,12 @@ export const getDateFormat = (
     return dateForm;
 };
 
+const isMomentInput = (value: unknown): value is MomentInput =>
+    typeof value === 'string' ||
+    typeof value === 'number' ||
+    value instanceof Date ||
+    isMoment(value);
```

Comment:
is this for fixing type erorrs? if yes, makes me think why it was not reporting...

Reply:
I kept getting this error, it wouldn't compile for me because that function accepts `MomentInput` but we're passing in `unknown` I'm also confused why this isn't getting picked up by others

Reply:
hmm, I see, it's strange...